optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -118,42 +117,34 @@ Each of these families of algorithms has multiple optimization algorithms:
 |    <<geneticAlgorithms,Genetic Algorithms>> |4/5 |3/5 |2/5 |5/5|Yes
 |===
 
-If you want to learn more about metaheuristics, read the free books http://www.cs.gmu.edu/~sean/book/metaheuristics/[Essentials of Metaheuristics] or http://www.cleveralgorithms.com/[Clever Algorithms].
+To learn more about metaheuristics, see  http://www.cs.gmu.edu/~sean/book/metaheuristics/[Essentials of Metaheuristics] or http://www.cleveralgorithms.com/[Clever Algorithms].
 
 
 [[whichOptimizationAlgorithmsShouldIUse]]
 == Which Optimization Algorithms Should I Use?
 
-The _best_ optimization algorithms configuration for your use case depends heavily on your use case.
-Nevertheless, this vanilla recipe will get you into the game with a pretty good configuration, probably much better than what you're used to.
+The best optimization algorithms configuration to use depends heavily on your use case.
+Nevertheless, this basic procedure will provide basic configuration.
 
-Start with a quick configuration that involves little or no configuration and optimization code:
+. Start with a quick configuration that involves little or no configuration and optimization code:
+ See <<firstFit,First Fit>>.
 
-. <<firstFit,First Fit>>
+. Next, implement <<planningEntityDifficulty,planning entity difficulty>> comparison and turn it into <<firstFitDecreasing,First Fit Decreasing>>.
 
-Next, implement <<planningEntityDifficulty,planning entity difficulty>> comparison and turn it into:
+. Next, add Late Acceptance behind it:
+.. First Fit Decreasing.
+.. <<lateAcceptance,Late Acceptance>>. A Late Acceptance size of 400 usually works well.
 
-. <<firstFitDecreasing,First Fit Decreasing>>
+At this point, the return on invested time lowers and the result is likely to be sufficient.
 
-Next, add Late Acceptance behind it:
-
-. First Fit Decreasing
-. <<lateAcceptance,Late Acceptance>>. A Late Acceptance size of 400 usually works well.
-
-At this point __the free lunch is over__.
-The return on invested time lowers.
-The result is probably already more than good enough.
-
-But you can do even better, at a lower return on invested time.
+However, this can be improved at a lower return on invested time.
 Use the <<benchmarker,Benchmarker>> and try a couple of different Tabu Search, Simulated Annealing and Late Acceptance configurations, for example:
 
-. First Fit Decreasing
-. <<tabuSearch,Tabu Search>>. An entity tabu size of 7 usually works well.
+. First Fit Decreasing: <<tabuSearch,Tabu Search>>. An entity Tabu Search size of seven is recommended.
 
-Use the <<benchmarker,Benchmarker>> to improve the values for those size parameters.
+Use the <<benchmarker,Benchmarker>> to improve the values for the size parameters.
```

Comment:
This sentence should still be there, even if we delete the size parameter info above, just to let them know they they can tweak the size parameter...